packages/dx-react-core/src/plugged/plugin-indexer.jsx

```diff
@@ -8,7 +8,7 @@ export const PluginIndexer = (
   { children },
   { positionContext },
 ) => (
-  <div style={{ display: 'none' }}>
+  <React.Fragment>
```

Comment:
What about `display: none`? Is it unnecessary here?

Reply:
Fragment is an virtual element. It will not be present in DOM.